tests/mpesa-rust/dynamic_qr_code_tests.rs

```diff
@@ -0,0 +1,18 @@
+use crate::get_mpesa_client;
+
+#[tokio::test]
```

Comment:
Rebase or merge the latest `master` branch to get updated testing crate, see current tests to see how you can implement tests for `dynamic_qr`